ep/build-velox/src/modify_velox.patch

```diff
@@ -139,6 +139,19 @@ index 2cabfc29a..54329ce23 100644
  
  add_library(
    velox_dwio_arrow_parquet_writer_test_lib
--- 
+
+diff --git a/velox/connectors/hive/storage_adapters/abfs/RegisterAbfsFileSystem.cpp b/velox/connectors/hive/storage_adapters/abfs/RegisterAbfsFileSystem.cpp
+index e2a638df6..e383cf205 100644
+--- a/velox/connectors/hive/storage_adapters/abfs/RegisterAbfsFileSystem.cpp
++++ b/velox/connectors/hive/storage_adapters/abfs/RegisterAbfsFileSystem.cpp
+@@ -38,7 +38,6 @@ std::shared_ptr<FileSystem> abfsFileSystemGenerator(
+
+ void registerAbfsFileSystem() {
+ #ifdef VELOX_ENABLE_ABFS
+-  LOG(INFO) << "Register ABFS";
```

Comment:
yes, the change only for making `vcpkg static build` bundle works, otherwise would break the run with Segmentation fault issue when calling Google::LOG::INIT during hive connector registration ，（non static build not need this change）.
  
I think (just guess) the issue caused due to gluten copy `HiveConnector.cpp.o` object to generate `libvelox.so` and then glob static lib content/symbols since we use static version glog.  However, when loading libvelox.so to runtime, the `HiveConnector` would directly call static `VELOX_REGISTER_CONNECTOR_FACTORY` to register hive connector and the also filesystems, and then call `LOG(INFO)` API if the register filesystem method have. However, the glob in that phase looks not ready to be use.

https://github.com/facebookincubator/velox/blob/main/velox/connectors/hive/HiveConnector.cpp#L237

```
Program received signal SIGSEGV, Segmentation fault.
0x0000555555574e30 in google::LogMessage::Init(char const*, int, int, void (google::LogMessage::*)()) ()
```

I have actually also can repdoduce the issue with a small code project, i can share if you need, currently this is the only way can fix the issue

Reply:
Just curious the code change. It's ok to me to work around it as the pr proposed.
@marin-ma, do you happen to know this issue?